fast64_internal/oot/oot_scene_room.py

```diff
@@ -172,13 +172,27 @@ def drawExitProperty(layout, exitProp, index, headerIndex, objName):
 
 class OOTObjectProperty(bpy.types.PropertyGroup):
     expandTab: bpy.props.BoolProperty(name="Expand Tab")
-    objectID: bpy.props.EnumProperty(items=ootEnumObjectID, default="OBJECT_HUMAN")
-    objectIDCustom: bpy.props.StringProperty(default="OBJECT_HUMAN")
+    objectKey: bpy.props.EnumProperty(items=ootData.objectData.ootEnumObjectKey, default="obj_human")
+    objectID: bpy.props.EnumProperty(items=ootData.objectData.ootEnumObjectIDLegacy, default="OBJECT_HUMAN")
+    objectIDCustom: bpy.props.StringProperty(default="OBJECT_CUSTOM")
 
+    @staticmethod
+    def upgrade_object(obj):
+        if obj.fast64.oot.version < 1:
+            print(f"Processing '{obj.name}'...")
+            ootData.objectData.upgradeObjectList(obj.ootRoomHeader.objectList)
+            ootData.objectData.upgradeAltHeaders(obj)
```

Comment:
I feel like `upgradeObjectList(obj.ootRoomHeader.objectList)` could just go in `upgradeAltHeaders`, which could be renamed to `upgradeRoomHeaders`